bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/AbstractBigtableTable.java

```diff
@@ -518,6 +520,24 @@ public void mutateRow(RowMutations rowMutations) throws IOException {
     }
   }
 
+  protected Result mutateRowResult(RowMutations rowMutations) throws IOException {
+    LOG.trace("mutateRow(RowMutation)");
+    if (rowMutations.getMutations().isEmpty()) {
+      return Result.EMPTY_RESULT;
+    }
+    Span span = TRACER.spanBuilder("BigtableTable.mutateRow").startSpan();
+    try (Scope scope = TRACER.withSpan(span)) {
+      FutureUtil.unwrap(clientWrapper.mutateRowAsync(hbaseAdapter.adapt(rowMutations)));
+      return Result.EMPTY_RESULT;
```

Comment:
@igorbernstein2 @mutianf should we try and return an actual result here? Any ideas on how we can do that given the client wrapper returns void?